datalad_ria/patches/sshremoteio.py

```diff
@@ -0,0 +1,83 @@
+"""
+
+This patch fixes the __init__ function of SSHRemoteIO, i.e.
+it adds error handling and a proper SSHRI-to-string function.
+"""
+
+import logging
+import subprocess
+
+from datalad import ssh_manager
+from datalad.support.exceptions import CommandError
+from datalad_next.patches import apply_patch
+
+# use same logger as -core
+lgr = logging.getLogger('datalad.customremotes.ria_remote')
+
+
+# This method is taken from
+# datalad@8a145bf432ae8931be7039c97ff602e53813d238
+# datalad/distributed/ore-remote.py:SSHRemoteIO.__init___
+
+
+DEFAULT_BUFFER_SIZE = 65536
```

Comment:
Using `datalad_next.utils.consts.COPY_BUFSIZE` might be a better choice than continuing to hardcode this value.

Reply:
I thought so too, but since the original code uses this constant value and because the code is difficult to navigate, I did not want to "upset" it needlessly.

Anyway, I changed it to our common constant in 55a4f351e6270d8f9dbe21c569d2e22848b1407c